BAZProject/Views/MovieDetail/CollectionViewCells/CastCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CastCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by Mario Arceo on 01/03/23.
+//
+
+import UIKit
+
+class CastCollectionViewCell: UICollectionViewCell {
+    
+    let identifier = "CastCustomCell"
```

Comment:
Para no crear una instancia de esta clase es mejor poner el identifier con constante estática.
```suggestion
    static let identifier = "CastCustomCell"
```